PRESIGNED-PAUSE.md

```diff
@@ -40,6 +42,19 @@ doesn’t even need to be funded. These are offchain signatures.
 
 A Facilitator will collect the signatures and execute the contract.
 
+### 1. Ensure no GAPs with PSPs coverage
+
+We need to ensure there is no GAPs in the PSPs coverage during upgrade.
```

Comment:
nit, can we change all stylizing of GAP to Gap/gap, since I think it's not an acronym in this context and you mean a literal gap